examples/react/css/src/hierarchical-menu.css

```diff
@@ -0,0 +1,73 @@
+/* HierarchicalMenu */
+.ais-HierarchicalMenu-list .ais-HierarchicalMenu-list {
+  margin-left: calc(var(--ais-spacing) * 1);
+  padding-left: calc(var(--ais-spacing-half) * 0.75);
+  border-left: 1px solid
+    rgba(var(--ais-border-color-rgb), var(--ais-border-color-alpha));
+}
+
+.ais-HierarchicalMenu-link {
+  color: rgba(var(--ais-text-color-rgb), 0.9);
+  display: flex;
+  align-items: center;
+  padding: calc(var(--ais-spacing-half) * 0.75) calc(var(--ais-spacing-half));
+  border-radius: calc(var(--ais-spacing) / 4);
+  transition: background-color var(--ais-transition-duration)
+      var(--ais-transition-timing-function),
+    color var(--ais-transition-duration) var(--ais-transition-timing-function);
+
+  &:before {
+    transition: transform var(--ais-transition-duration)
+      var(--ais-transition-timing-function);
+    background-image: url(data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2024%2024%22%20fill%3D%22none%22%20stroke%3D%22%23262626%22%20stroke-width%3D%222%22%20stroke-linecap%3D%22round%22%20stroke-linejoin%3D%22round%22%3E%3Cpolyline%20points%3D%226%209%2012%2015%2018%209%22%3E%3C%2Fpolyline%3E%3C%2Fsvg%3E);
```

Comment:
I had to duplicate the SVG several times because we can't use `currentColor` for the stroke, so I had to change the stroke color for when it's selected and also for dark mode.

Reply:
Can we have an internal variable?

Reply:
Good idea! I also added a comment at the top of each one to let us know what color it uses.
And I updated the `stroke-width` to `1.6`.